api/src/pcapi/routes/native/v1/account.py

```diff
@@ -278,6 +278,13 @@ def create_account_with_google_sso(body: serializers.GoogleAccountRequest) -> au
     except exceptions.UnderAgeUserException:
         raise api_errors.ApiErrors({"dateOfBirth": "The birthdate is invalid"})
 
+    try:
+        dms_subscription_api.try_dms_orphan_adoption(created_user)
+    except Exception:  # pylint: disable=broad-except
+        logger.exception(
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** On pourrait presque factoriser le try:catch + log dans une fonction pou éviter de dupliquer

Very low logaf, parce que ça a le mérite d'être explicite
C'est toi qui voit 😄 ‏